packages/rocketchat-lib/server/functions/updateMessage.js

```diff
@@ -4,55 +4,53 @@ RocketChat.updateMessage = function(message, user, originalMessage) {
 	if (!originalMessage) {
 		originalMessage = RocketChat.models.Messages.findOneById(message._id);
 	}
-
-	// For the Rocket.Chat Apps :)
-	if (message && Apps && Apps.isLoaded()) {
-		const appMessage = Object.assign({}, originalMessage, message);
-
-		const prevent = Promise.await(Apps.getBridges().getListenerBridge().messageEvent('IPreMessageUpdatedPrevent', appMessage));
-		if (prevent) {
-			throw new Meteor.Error('error-app-prevented-updating', 'A Rocket.Chat App prevented the message updating.');
+	if (originalMessage.msg !== message.msg) {
```

Comment:
instead of shifting everything to inside the `if`, can you please change it to something like:
```javascript
if (originalMessage.msg !== message.msg) {
	return;
}
/* rest of the code */
```
this way the code looks nicer and clean as so the diff :)

Reply:
Yeah true, I'll do that but shouldn't it be,
```javascript
if (originalMessage === message.msg) {
       return;
}
```
because we want to skip everything and return if both messages are equal ?

Reply:
Yes, that's right :)